### ISSUE TITLE
Info - Issues with TypeScript and production builds

### ISSUE THREAD

**Issue by KFlash** [meriyah/meriyah#30]
FYI! I have recently experienced multiple issues where the TypeScript compiler bundle the Meriyah source code wrong. 

This is very hard for me to catch because all test cases pass as normal and the code works as normal - see CircleCI.

I only notice this if some code doesn't work in the REPL. 

Most common issues

- `const enum` values for tokens
- `const enum` values for bitmasks

If you are using Meriyah and encounter an issue related to this. Don't hesitate to let me know and I will bundle it again and make sure it works.

cc / @aladdin-add @axefrog @nchanged

**Comment by KFlash** [comment]
I used 2 hours to track one stupid issue, so to explain better.

`Token.QuestionMarkPeriod` have value `67108988`.  When bundled only in the lexer code that token had that value. **Same token** in `parseMemberOrUpdateExpression` had value `67108989`. 
The value had increased with `+1`.   That broke most of the `ES next` implementation.

Dunno how it got fixed, but I tried to trick the TS compiler and finally got it working.
